core/src/i18n/en.yaml

```
@@ -132,6 +138,8 @@ applications:
     namespace-bindings: Namespace Bindings
     provided-services: Provided Services and events
   title: Applications
+  tooltips:
+    provisioning: The Application is being provisioned asynchronously.
```

```suggestion
    provisioning: The application is being provisioned asynchronously.
```
If you mean our Kyma Application (https://kyma-project.io/docs/kyma/latest/05-technical-reference/00-custom-resources/ac-01-application) then it should be uppercase, if not then let's stick to lowercase